app/models/util/file_manager.rb

```diff
@@ -48,6 +50,14 @@ def flat_files_directory
       end
     end
 
+    def beta_flat_files_directory
```

Comment:
for the dump file we have the method `pg_dump_file(schema='')`, we pass the schema it might be better to add the schema to this existing method so that we follow the same pattern. Any method that tells us the file location or directory, we pass the schema. 

This also helps clean up conditionals in the calling method.